index.bs

```diff
@@ -94,6 +94,12 @@ spec: WebXR Hit Test Module; urlPrefix: https://immersive-web.github.io/hit-test
     background-repeat: repeat;
     background-color: #282828;
   }
+  @media (prefers-color-scheme: light) {
+    .unstable {
+      background-image: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' width='300' height='290'><text transform='rotate(-45)' text-anchor='middle' font-family='sans-serif' font-weight='bold' font-size='70' y='210' opacity='.1'>Unstable</text></svg>");
```

Comment:
Do we need to specify the same background-image as the one for the dark mode? Seems that the rest of the CSS properties get inherited in the light mode (i.e. we're missing on background-repeat and yet the graphics still repeats).

Reply:
Good catch! I forgot to update the style for the background image in dark mode.
Otherwise it's not visible.